layers/+vim/vinegar/keybindings.el

```diff
@@ -1,4 +1,3 @@
-(require 'dired-x)
```

Comment:
I'm using the `develop` branch and if I have vinegar enabled I get an error that `dired-omit-mode` is void with this line deleted.

If I add the line back in everything works as expected, so it seems as though the `dired-x` module isn't being loaded by the time the `vinegar` layer is loaded.